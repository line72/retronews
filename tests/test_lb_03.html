<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</p>
<ul>
<li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</li>
<li>Lorem:
<ul>
<li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</li>
<li>Lorem ipsum dolor sit amet, consectetur.</li>
<li>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</li>
</ul>
</li>
</ul>
<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</p>
