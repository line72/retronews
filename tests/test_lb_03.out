Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur
adipiscing elit, sed do eiusmod tempor

- Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
  eiusmod tempor.
- Lorem:

  - Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
    eiusmod tempor.
  - Lorem ipsum dolor sit amet, consectetur.
  - Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
    eiusmod tempor.

Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
eiusmod tempor. Lorem ipsum dolor sit amet, consectetur adipiscing
elit, sed do eiusmod tempor. Lorem ipsum dolor sit amet, consectetur
adipiscing elit, sed do eiusmod tempor